src/main/java/com/exasol/adapter/dialects/oracle/OracleQueryRewriter.java

```diff
@@ -43,16 +41,15 @@ private boolean isGenerateJdbcDatatypeMappingForOCIEnabled() {
         return this.properties.isEnabled(OracleProperties.GENERATE_JDBC_DATATYPE_MAPPING_FOR_OCI_PROPERTY);
     }
 
-
     @Override
     protected String generateImportStatement(String connectionDefinition, String pushdownQuery) throws SQLException {
         return generateImportStatement(connectionDefinition, null, pushdownQuery);
     }
 
     @Override
     protected String generateImportStatement(final String connectionDefinition,
-                                             List<DataType> selectListDataTypes, final String pushdownQuery) {
-        if (isGenerateJdbcDatatypeMappingForOCIEnabled()) {
+            final List<DataType> selectListDataTypes, final String pushdownQuery) {
+        if (isGenerateJdbcDatatypeMappingForOCIEnabled() && !selectListDataTypes.isEmpty()) {
             final String columnDescription = this.createImportColumnsDescription(selectListDataTypes);
             return "IMPORT INTO (" + columnDescription + ") FROM ORA " + connectionDefinition + " STATEMENT '" + pushdownQuery.replace("'", "''") + "'";
         } else {
```

Comment:
The additional condition avoids rendering invalid syntax `IMPORT INTO () FROM ORA` for an empty `selectListDataTypes`.